lib/stdlib/src/sofs.erl

```diff
@@ -327,10 +407,24 @@ from_sets(_) ->
     erlang:error(badarg).
 
 -doc """
-Equivalent to [`relation(Tuples, Type)`](`relation/2`) where `Type` is the size
-of the first tuple of `Tuples` is used if there is such a tuple.
+Equivalent to [`relation(Tuples, Type)`](`relation/2`), where `Type` is the size
+of the first tuple of `Tuples`, if such a tuple exists.
 
 If tuples is `[]`, then `Type` is `2`.
+
+## Examples
+
+```erlang
+1> sofs:type(sofs:relation([{1,a},{1,b}])).
```

Comment:
Should this example also have a `sofs:to_external/1` variant?

Reply:
Yes, that could make it clearer. Added.